media/javascript/underscore.js

```diff
@@ -1 +1 @@
-../static/vendor/underscore-standalone.js
\ No newline at end of file
+../../readthedocs/static/vendor/underscore-standalone.js
```

Comment:
I know that you just updated the path of the link, but why we are adding static files from our app into the media directory? Shouldn't this be served from the `STATICFILES_DIRS`?

Reply:
There's two problems:

* Firstly, `collectstatic` actually checks that every symlink resolves. Since `/media/static` won't exist if this PR is merged, any symlink referring to there needs updating.
* The second problem is that some already built docs refer to paths that are referenced by symlinks. For example, `<script type="text/javascript" src="https://media.readthedocs.org/javascript/underscore.js"></script>` which is in all the built docs relies on this symlink.